content/concepts/consensus-network/transaction-malleability.md

```diff
@@ -86,35 +72,29 @@ For greater security, these guidelines provide multiple layers of protection.
 
 If the software you use to interface with the XRP Ledger sends malleable transactions, a malicious actor may be able to trick your software into losing track of a transaction's final outcome and potentially (in the worst case) sending equivalent payments multiple times.
 
-If you use single-signatures and always enable the `tfFullyCanonicalSig` flag, you are not vulnerable to this exploit. If you use multi-signatures, you may be vulnerable if you or your signers provide more signatures than necessary.
+If you use single-signatures only, you are not vulnerable to this exploit. If you use multi-signatures, you may be vulnerable if you or your signers provide more signatures than necessary.
 
 ### Exploit Scenario Steps
 
 The process to exploit a vulnerable system follows a series of steps similar to the following:
 
-1. The vulnerable system constructs and signs a transaction without enabling `tfFullyCanonicalSig`.
-
-    Three ways that a transaction may not enable the `tfFullyCanonicalSig` flag are:
-
-    - The system explicitly specifies a `Flags` field that does not have the `tfFullyCanonicalSig` bit enabled.
-    - The transaction is multi-signed and does not explicitly enable the `tfFullyCanonicalSig` flag.
-    - The system omits the `Flags` field from the transaction fields, but uses a non-standard implementation that does not automatically enable `tfFullyCanonicalSig` when signing.
+1. The vulnerable system constructs a multi-signed transaction and collects more than the necessary number of signatures.
 
-    To be vulnerable, the transaction must be signed with an ECDSA key pair. If multi-signed, the transaction must be signed by at least one ECDSA key pair.
-
-    Most likely, the vulnerable transaction uses a fully-canonical signature, but the flags indicate that the transaction would also be valid with a non-fully-canonical one. The transaction may also use `LastLedgerSequence` so that its final outcome is clear in a finite amount of time.
+    If an authorized signer is malicious or irresponsible, the transaction could also be vulnerable if that signer's signature is not included but could be added.
 
 2. The system notes the identifying hash of the vulnerable transaction, submits it to the XRP Ledger network, then begins monitoring for that hash to be included in a validated ledger version.
 
 3. A malicious actor sees the transaction propagating through the network before it becomes confirmed.
 
-4. The malicious actor calculates the alternate signature for the vulnerable transaction.
+4. The malicious actor calculates removes an extra signature from the vulnerable transaction.
```

Comment:
@mDuo13 , just curious why this bullet is "The x actor" and the one above is "A x actor".

Reply:
Step 3 is the first time a malicious actor is part of the process, so it's "a" malicious actor. In step 4, I'm referring to the same malicious actor, so it's "the" malicious actor.

Reply:
Awkward phrasing: "calculates removes".

Makes more sense as "The malicious actor removes an extra signature..."